File: Assets/kolmich/KGFCameraSystem/demo/scripts/SimpleRotate.js

#pragma strict

public var speed : Vector3;

function OnBecameVisible () {
	enabled = true;	
}

function OnBecameInvisible () {
	enabled = false;	
}

function Update () {
	transform.Rotate(Time.deltaTime * speed.x, Time.deltaTime * speed.y, Time.deltaTime * speed.z);
}

File: Assets/kolmich/KGFCameraSystem/demo/scripts/SimpleRotate.cs
using UnityEngine;
using System.Collections;

public class SimpleRotate : MonoBehaviour
{
	public Vector3 itsRotate;
	
	private Vector3 itsCurrentRotation;
	void Update ()
	{
		itsCurrentRotation.x += itsRotate.x/360.0f * Time.deltaTime;
		itsCurrentRotation.y += itsRotate.y/360.0f * Time.deltaTime;
		itsCurrentRotation.z += itsRotate.z/360.0f * Time.deltaTime;
		gameObject.transform.localRotation = Quaternion.Euler(itsCurrentRotation);
	}
}
